### PR TITLE
Compiler warnings part 1

### PR DESCRIPTION
**Pull request by antares1470**
### Description of the Change
Removed a majority of the compiler warnings from the following modules:
- Core Analytic View
- Core Arrangement Plugins
- Core Attribute Calculator View
- Core Attribute Editor View

### Benefits
Less compiler warnings looks a lot nicer in the build logs and may help with performance

### CODE REVIEW

File: CoreAnalyticView/src/au/gov/asd/tac/constellation/views/analyticview/AnalyticConfigurationPane.java [hunk 1]
```diff
@@ -143,7 +143,7 @@ public AnalyticConfigurationPane() {
                 .map(plugin -> plugin.getPlugin().getResultType())
                 .collect(Collectors.toSet()).size() <= 1) :
                 "A category should be populated only with analytics of the same result type.";
-        this.categoryList = new ListView();
+        this.categoryList = new ListView<String>();
```
Comment (cygnus-x-1) [inline CoreAnalyticView/src/au/gov/asd/tac/constellation/views/analyticview/AnalyticConfigurationPane.java:146]: I'd assume this can just be `ListView<>`?

Comment (antares1470) [reply CoreAnalyticView/src/au/gov/asd/tac/constellation/views/analyticview/AnalyticConfigurationPane.java:146]: Yep will be fixed in the next commit

File: CoreAnalyticView/src/au/gov/asd/tac/constellation/views/analyticview/translators/AnalyticTranslator.java [hunk 2]
```diff
@@ -41,5 +41,6 @@ public void setResult(final R result) {
 
     public abstract String getName();
 
+    @SuppressWarnings("rawtypes") //taw type needed for AnyToReport implementation
```
Comment (arcturus2) [inline CoreAnalyticView/src/au/gov/asd/tac/constellation/views/analyticview/translators/AnalyticTranslator.java:44]: "raw"

File: CoreArrangementPlugins/src/au/gov/asd/tac/constellation/plugins/arrangements/planar/PQTree.java [hunk 3]
```diff
@@ -31,25 +33,22 @@
...
     }
 
     public PQTree(final int numberNodes) {
         currentNumber = 1;
         this.root = new PQNode(NodeType.PNODE);
-        leaves = new Set[numberNodes];
-        for (int i = 0; i < numberNodes; i++) {
-            leaves[i] = new HashSet<>();
-        }
+        leaves = new ArrayList<>(Collections.nCopies(numberNodes, new HashSet<>()));
```
Comment (arcturus2) [inline CoreArrangementPlugins/src/au/gov/asd/tac/constellation/plugins/arrangements/planar/PQTree.java:51]: 👍

**Pull request merged**